lerna.json

```diff
@@ -9,8 +9,10 @@
         "**/*.spec.ts",
         "**/*.test.ts"
       ],
-      "message": "chore(release): publish new package versions [skip ci]",
-      "allowBranch": ["master", "release-*"]
+      "message": "chore(release): version bump [skip ci]",
```

Comment:
nit: better to keep the original message I think because there are multiple packages, not a single version bump